src/elm/Session/LoggedIn.elm

```diff
@@ -362,11 +352,23 @@ viewHelper thisMsg page profile_ ({ shared } as model) content =
             Hidden ->
                 text ""
         , div [ class "flex-grow" ]
-            [ if isContentAllowed then
-                content
+            [ case model.hasKyc of
+                FeatureLoading ->
+                    div [ class "full-spinner-container h-full" ]
+                        [ div [ class "spinner spinner--delay mt-8" ] [] ]
```

Comment:
I didn't use the `Page.fullPageLoading` here because importing `Page` module caused circular dependencies. I think it's fine to have it like this.